.envrc

```diff
@@ -238,4 +237,4 @@ EOF
 fi
 
 # Since we can't use an EXIT routine we need to guarantee we delete the file here
-rm "$_SENTRY_LOG_FILE"
+test -f "${_SENTRY_LOG_FILE}" && rm "$_SENTRY_LOG_FILE"
```

Comment:
You can just use `rm -f` instead of this. It won't fail if it doesn't exist.

Reply:
Oh, in fact, this doesn't even work, since `test` will fail and direnv will exit 1. But `rm -f` does.